notebook/tree/handlers.py

```diff
@@ -49,6 +50,7 @@ def get(self, path=''):
                 notebook_path=path,
                 breadcrumbs=breadcrumbs,
                 terminals_available=self.settings['terminals_available'],
+                server_root=os.path.basename(os.path.normpath(self.settings['server_root_dir'])),
```

Comment:
Ok, so should it look like this?

![image](https://cloud.githubusercontent.com/assets/512354/23727437/e795a0de-040c-11e7-99fc-d15f057fbdef.png)